lib/register.js

```diff
@@ -1,26 +1,4 @@
 'use strict';
```

Comment:
I think we might want to do a major bump on these changes either way, just to be safe.

Before, it only broke on pushgateway, which is not too usual to use, and it only brok on something polyfillable. Now we break on syntax. WDYT?